File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_list/view/CourseListView.java
package org.wikiedufoundation.wikiedudashboard.course_list.view;

import org.wikiedufoundation.wikiedudashboard.course_list.data.ExploreCoursesResponse;
import org.wikiedufoundation.wikiedudashboard.helper.Progressive;
import org.wikiedufoundation.wikiedudashboard.helper.Toaster;

public interface CourseListView extends Progressive, Toaster {
    void setData(ExploreCoursesResponse data);
}

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_list/view/CourseListView.kt
package org.wikiedufoundation.wikiedudashboard.course_list.view

import org.wikiedufoundation.wikiedudashboard.course_list.data.ExploreCoursesResponse
import org.wikiedufoundation.wikiedudashboard.helper.Progressive
import org.wikiedufoundation.wikiedudashboard.helper.Toaster

interface CourseListView : Progressive, Toaster {
    fun setData(data: ExploreCoursesResponse)
}
